bridge/jira/export.go

```diff
@@ -101,16 +101,22 @@ func (je *jiraExporter) cacheAllClient(ctx context.Context, repo *cache.RepoCach
 			continue
 		}
 
-		user, err := repo.ResolveIdentityImmutableMetadata(metaKeyJiraLogin, login)
-		if err == identity.ErrIdentityNotExist {
-			continue
+		explicitIdentity, ok := cred.GetMetadata(auth.MetaKeyIdentity)
```

Comment:
So, you associate the credential to an identity if the credential has been tagged with an identity metadata. This metadata was set in the version 0.6.0 when I wandered (incorrectly) towards this kind of tagging. They are not set anymore so it would only work as a "legacy" support, not as a solution onward.

Honestly I'm torn on this. Yes it's nice to stay compatible even for temporary mistake but at the same time git-bug is at the stage where it's ok to have light breaking changes instead of accumulating tech debt. Bridges are part of those as you just have to delete the config and set it up again. I understand your problem is a bit different but it seems to me that the correct solution would the the ones mentioned in #336 and #335 

Does that sounds correct to you ?

Reply:
>  I understand your problem is a bit different but it seems to me that the correct solution would the the ones mentioned in #336 and #335

Well, as you mentioned in #351:

> This would solve this problem but it's not perfect either. I think we might also need a way to either:
>
>associate an existing credential to an existing user
>tag an existing user with a login for a specific bridge

This is one way to associate an existing credential to an existing user. If the credential metadata is scheduled for removal then we can push the identity field into the main credential config instead of the meta config heading right? 

The alternative solution of tagging an existing user with a login for a specific bridge is fine too, but it sounds more complicated to implement and simple is nice. 

I'm happy with either solution. I also wouldn't mind if you take this solution for now and replace it with a better one later.

Reply:
> If the credential metadata is scheduled for removal then we can push the identity field into the main credential config instead of the meta config heading right?

It's not scheduled for removal. The wrong path I went in is to tag the credential with the identity ID. It lead to a bunch of problems because there is multiple scenario when configuring a bridge: you already have your identity, your identity will be imported by the bridge, ... It's a mess and doesn't work out. The correct way it seems is to tag both credential and identity with the bridge login and match them later.

My impression at the moment is that we just need 1) the missing command for the login/password credential and 2) some extra capability to manipulate those tags for the complex cases. I could be convinced otherwise though.